### PR TITLE
Migration cypress to playwright - rich text editor

### PR DESCRIPTION
**Pull request by Ben0189**
Related PBI: https://github.com/tinacms/tinacms/issues/4580

 Migrate cypress to playwright (first stage)

Test cover 
- Rich text editor parsing

### CODE REVIEW

File: cypress.config.ts
Comment (calumjs) [inline cypress.config.ts:1]: Do you still need this?

File: playwright/tina-playwright/tests/rich-text-editor/rich-text-editor-parsing.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    const iframe = page.frameLocator('iframe[data-test="tina-iframe"]');
+
+    // Locate the <em> tag inside the iframe content and check if it contains the correct text
+    const italicElement = iframe.locator("em");
+
+    // Assert that the italic element exists
+    await expect(italicElement).toBeVisible();
+
+    // Assert that the italic element contains the correct text
+    await expect(italicElement).toHaveText("Italic");
```
Comment (calumjs) [inline playwright/tina-playwright/tests/rich-text-editor/rich-text-editor-parsing.spec.ts:42]: Add more test cases e.g. within a word, using underscores, etc.

**Review by calumjs — COMMENTED** [review]
This 1 test looks good, but needs more tests

File: playwright/tina-playwright/tests/rich-text-editor/rich-text-editor-parsing.spec.ts (hunk 1 above)
Comment (Ben0189) [reply playwright/tina-playwright/tests/rich-text-editor/rich-text-editor-parsing.spec.ts]: Done

File: cypress.config.ts
Comment (Ben0189) [reply cypress.config.ts:1]: will be remove once all test is migrated

File: cypress.config.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from 'cypress'
```
Comment (JackDevAU) [inline cypress.config.ts:1]: Do we need this file?

Comment (Ben0189) [reply cypress.config.ts]: Done removed

**Review by JackDevAU — APPROVED** [review]
🦙

**Pull request merged**